### ISSUE TITLE
I got very high and non-decrease RPN_bbox_loss 

### ISSUE THREAD

**Issue by datnguyenzzz** [matterport/Mask_RCNN#2371]
Hi everyone !

I'm trying to fine-tune coco pre-trained model in object detection task (without masks layers) . I used same configs as authors used in their blog. But I got very high rpn bounding box loss and non-decrease after training done with learning rate~1e-6. Loss always around 240. If I set it higher, rpn_bbox_loss would becoming NaN after few epochs, the other loss seems not that bad. 

410/1000 [===========>..................] - ETA: 29:53 - loss: 245.9679 - rpn_class_loss: 0.0386 - rpn_bbox_loss: 244.3961 - 
411/1000 [===========>..................] - ETA: 29:49 - loss: 245.5678 - rpn_class_loss: 0.0385 - rpn_bbox_loss: 243.9965 - mrcnn_class_loss: 0.3495 - mrcnn_bbox_loss: 0.0633

Is there any good advice? Many thanks

**Comment by konstantin-frolov** [comment]
Hi. How do you create the bbox? MRCNN creates target bbox from target mask on the fly. If you don't have a mask, rcnn won't be able to create a target bbox and find any bbox.

**Comment by kecyarchana** [comment]
> Hi everyone !
> 
> I'm trying to fine-tune coco pre-trained model in object detection task (without masks layers) . I used same configs as authors used in their blog. But I got very high rpn bounding box loss and non-decrease after training done with learning rate~1e-6. Loss always around 240. If I set it higher, rpn_bbox_loss would becoming NaN after few epochs, the other loss seems not that bad.
> 
> 410/1000 [===========>..................] - ETA: 29:53 - loss: 245.9679 - rpn_class_loss: 0.0386 - rpn_bbox_loss: 244.3961 - 411/1000 [===========>..................] - ETA: 29:49 - loss: 245.5678 - rpn_class_loss: 0.0385 - rpn_bbox_loss: 243.9965 - mrcnn_class_loss: 0.3495 - mrcnn_bbox_loss: 0.0633
> 
> Is there any good advice? Many thanks

Hello, could you solve this issue? I am having the same problem with rpn_bbox_loss. Even with using coco pretrained weights my loss starts with high number like 20 and decreases by only decimal ?  Any suggestions to make it better ?

**Comment by f-macfarlane95** [comment]
@kecyarchana did you ever solve this, I think I am having a similar issue. Thanks!
